src/alire/alire-vcss-git.adb

```diff
@@ -260,6 +269,30 @@ package body Alire.VCSs.Git is
       end if;
    end Status;
 
+   -------------------------
+   -- Transform_To_Public --
+   -------------------------
+
+   function Transform_To_Public (Remote : String) return URL is
```

Comment:
It would be good to have unit testing for this kind of functions...

Reply:
We have a custom setup at https://github.com/alire-project/alire/blob/master/src/alire/alire-selftest.adb, should we continue to use that or should we move to some real unit testing framework?

Reply:
Putting in place a unit testing framework would be better, but let's use that for now :+1: